ide/ui/ide-monaco/src/main/resources/META-INF/dirigible/ide-monaco/js/editor.js

```diff
@@ -605,6 +611,54 @@ function getModuleImports(text) {
     return moduleImports;
 }
 
+function setupPrettier(fileName) {
+    let editorApi = "/services/v4/ide/editor/prettier/config";
+    let prettierConfigRequest = `${editorApi}${fileName}`
+
+    fetch(prettierConfigRequest, {
+        method: "GET"
+    })
+    .then(function(response) { return response.json(); })
+    .then(function(configJson) {
+        registerPrettierFormatter(configJson, "javascript");
+    })
+    .catch((error) => {
+        let defaultConfigJson = {
+              "trailingComma": "es5",
+              "tabWidth": 4,
+              "semi": false,
+              "singleQuote": true,
+        };
+        registerPrettierFormatter(defaultConfigJson, "javascript");
+    });
+}
+
+function registerPrettierFormatter(configJson, language) {
+    configJson["plugins"] = [parserBabel];
```

Comment:
Do not access properties from the object like that. Use the dot notation: `configJson.plugins`